autoPlay.user.js

```diff
@@ -1170,6 +1171,20 @@ function useTreasureIfRelevant() {
 	}
 }
 
+function useMaxElementalDmgIfRelevant() {
+	// Check if Max Elemental Damage is purchased
+	if (hasItem(ITEMS.MAXIMIZE_ELEMENT)) {
+		if (isAbilityCoolingDown(ITEMS.MAXIMIZE_ELEMENT) || !canUseItem(ITEMS.MAXIMIZE_ELEMENT) || isAbilityActive(ITEMS.MAXIMIZE_ELEMENT)) {
+			return;
+		}
+		else {
+			// Max Elemental Damage is purchased, cooled down, and needed. Trigger it.
```

Comment:
Please use `getActiveAbilityLaneCount` to check for max elemental already active in current lane.